_acs-aem-commons/features/packagers/asset-packager/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: acs-aem-commons_feature
+title: Asset Packager
+description: Easily package pages and the associated assets
+redirect_from: /acs-aem-commons/features/asset-packager.html
+date: 2019-05-20
+feature-tags: administration
+initial-release: 4.2.0
+---
+
+# Purpose
+
```

Comment:
Is it standard to manually break lines at 80 char in README files?  If so, please update accordingly.

Reply:
I was just following suit with the other packagers with the formatting. I can update that